package-lock.json

```diff
@@ -102,6 +102,7 @@
                     "resolved": "https://registry.npmjs.org/extend-shallow/-/extend-shallow-2.0.1.tgz",
```

Comment:
I wouldn't have thought there should be this many changes to the package-lock. did you do an `npm install`?